code/modules/mob/dead/observer/spells.dm

```diff
@@ -59,4 +63,7 @@ var/global/list/boo_phrases=list(
 				A:spookymode=1
 
 			if(istype(A, /obj/machinery/ai_status_display))
-				A:spookymode=1
\ No newline at end of file
+				A:spookymode=1
+
+	if (shadowling_blocked_light_flicker)
+		to_chat(action.owner.client, "The nearby shadowling laughs at your unsuccessful attempt to flicker lights in it's presence.")
```

Comment:
http://www.elearnenglishlanguage.com/blog/english-mistakes/its/
Grammar. It's vs its.